specs/commandline/verify.md

```diff
@@ -173,6 +190,48 @@ Warning:  Always verify the artifact using digest(@sha256:...) rather than a tag
 Successfully verified signature for localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
 
+### Verify an arbitrary file located in OCI-compliant registry
+A user wants to verify a file stored as an OCI artifact in an OCI-compliant registry.
+```shell
+# Prerequisites: Signatures are stored in the registry referencing the file artifact
+
+# Use flag "--file" to enable verifying a file
+notation verify --file localhost:5000/myFile@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+```
+
+An example of output messages for a successful verification:
+
+```text
+Successfully verified signature for localhost:5000/myFile@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+```
+
+### Verify an arbitrary file located in file system
+A verifier wants to verify a file against its signatures located in file system.
+
+Trust policy to be used follows the rule below:
+1. User MAY pass in a trust policy scope via `--scope` flag. The value MUST follow Notation's trust policy [spec](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#registry-scopes-constraints). If the user specified trust policy does not exist in Notation's `trustpolicy.json` (use command `notation policy show` to check existence), then
+the [global trust policy](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#registry-scopes-constraints) is used.
+2. If user ignores the `--scope` flag, then the [global trust policy](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#registry-scopes-constraints) is used as default.
+```shell
+# Prerequisites: Both target file and signatures are stored in user's file system
+
+# Use flag "--file" to enable verifying a file
+# Use flag "--signature" to speicfy path where the signatures are stored
+# The global trust policy is used by default
+notation verify --file --signature ./mySignature1.sig --signature ./mySignature2.sig ./myFile.txt
+
+export NOTATION_EXPERIMENTAL=1
```

Comment:
Why do we want this to be experimental?

Reply:
Because `--scope` is an existing flag in the `notation v1.0.0 verify command`, and it is marked as `experimental` at the moment.

Reply:
@priteshbandi Yes, you are right, I'm not marking this new feature as experimental. Only the flag `--scope` is experimental here.

Reply:
So does that mean initially we will only support wildcard trust policy?

Reply:
@priteshbandi We will also support user specified trust policy, I'm working on specs in the specification repo to make this concept clear.